Dockerfile

```diff
@@ -67,6 +67,10 @@ RUN yum install -y --disableplugin=subscription-manager hostname procps \
     && rm -rf /var/cache/yum \
     && rm -rf /var/chef \
     && yum clean all
+    
+# download and install openjdk 8
+RUN curl -O https://vault.centos.org/8.3.2011/AppStream/x86_64/os/Packages/java-1.8.0-openjdk-headless-1.8.0.282.b08-2.el8_3.x86_64.rpm
+RUN yum localinstall -y --disableplugin=subscription-manager java-1.8.0-openjdk-headless-1.8.0.282.b08-2.el8_3.x86_64.rpm
```

Comment:
In general, combining multiple shell commands into one RUN cmd reduces the final image size as each RUN creates a new layer.  https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#minimize-the-number-of-layers
Not sure of the practical impact.  

Also, I think, this command would leave the jdk rpm in the image, so appending a rm would reduce the final image size a bit more.

Reply:
https://github.com/sonatype/docker-nexus3/pull/129/commits/7eb975d04fee774c5062fde56d8e26c6089a501c